boards/nxp/imx943_evk/imx943_evk_mimx94398_m7_0.dts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_imx943_m7_0.dtsi>
+#include "imx943_evk-pinctrl.dtsi"
+
+/ {
+	model = "NXP i.MX943 EVK board";
+	compatible = "nxp,imx943_evk";
+
+	chosen {
+		/* TCM */
```

Comment:
This comment may not be necessary, since lines below are self explanatory

Reply:
> This comment may not be necessary, since lines below are self explanatory

Sure, will delete it.